packages/components/psammead-story-promo-list/src/index.jsx

```diff
@@ -12,22 +12,36 @@ import {
   GEL_GROUP_4_SCREEN_WIDTH_MIN,
 } from '@bbc/gel-foundations/breakpoints';
 
-export const StoryPromoLi = styled.li.attrs({
+const promoListPropTypes = {
+  children: node.isRequired,
+  border: bool,
+};
+
+const promoListDefaultProps = {
+  border: true,
+};
+
+export const StoryPromoLiBase = styled.li.attrs({
   role: 'listitem',
 })`
+  ${({ border }) =>
+    border &&
+    `
+    @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+      border-bottom: 0.0625rem solid ${C_LUNAR};
+    }
+  `}
 
-  @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
-    border-bottom: 0.0625rem solid ${C_LUNAR};
+  &:last-child {
+    border: none;
   }
```

Comment:
This should really be part of the block that gets added when border styles are set.

```js
${({ border }) =>
  border &&
  `
  @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
    border-bottom: 0.0625rem solid ${C_LUNAR};
  }
  &:last-child {
    border: none;
  }
`}
```